docs/users_guide/framework_basics/dimensionless_quantities.md

```diff
@@ -160,6 +160,7 @@ with dimensionless quantities:
 ```cpp
 inline constexpr struct percent : named_unit<"%", mag<ratio{1, 100}> * one> {} percent;
 inline constexpr struct per_mille : named_unit<basic_symbol_text{"‰", "%o"}, mag<ratio(1, 1000)> * one> {} per_mille;
+inline constexpr struct ppm : named_unit<"ppm", mag<ratio(1, 1'000'000)> * one> {} ppm;
```

Comment:
```suggestion
inline constexpr struct parts_per_million : named_unit<"ppm", mag<ratio(1, 1'000'000)> * one> {} parts_per_million;
```

Reply:
That is a good idea, and thanks for the reference. However, as we do not have a `unit_symbols` namespace for those units, I think we may need to expose both terms:

```suggestion
inline constexpr struct parts_per_million : named_unit<"ppm", mag<ratio(1, 1'000'000)> * one> {} parts_per_million;
inline constexpr auto ppm = parts_per_million;
```